system/HTTP/IncomingRequest.php

```diff
@@ -375,6 +375,29 @@ public function negotiate(string $type, array $supported, bool $strictMatch = fa
         throw HTTPException::forInvalidNegotiationType($type);
     }
 
+    /**
+     * Checks if this request is.
+     *
+     * @param string $value HTTP verb or 'json'
+     * @phpstan-param string|'get'|'post'|'put'|'delete'|'head'|'patch'|'options'|'json' $value
+     */
+    public function is(string $value): bool
+    {
+        $value = strtolower($value);
+
+        $httpMethods = ['get', 'post', 'put', 'delete', 'head', 'patch', 'options'];
+
+        if (in_array($value, $httpMethods, true)) {
+            return strtolower($this->getMethod()) === $value;
+        }
+
+        if ($value === 'json') {
```

Comment:
Not sure if `json` belong here. But if we will have it, why not also have `ajax`? Basically the same story.

Reply:
I'm fine to add `ajax`.

Reply:
Added.

Reply:
We already have the isAJAX() method.

Reply:
Yes, we have, and as you may notice, we use it. IMO proposed `is()` method would serve as a convenient shortcut, so I don't see a problem with another way of accessing the same method.